### PR TITLE
Add Status section

### PR DESCRIPTION
**Pull request by sideshowbarker**
This needs be moved to the boilerplate section of the Bikeshed repo; but adding it here now in the interest of un-blocking the Geolocation spec transition.

### CODE REVIEW

**Comment by sideshowbarker** [comment]
@xfq After merging, autopublishing to TR seems to be failing with the same errors:

https://github.com/w3c/webappsec-secure-contexts/runs/3532134463#step:3:555

Since I’ve not gotten the boilerplate into the Bikeshed repo yet, I overrode it and put the status paragraphs into the source directly. And that works fine locally. But maybe not with spec-prod?

**Comment by sideshowbarker** [comment]
@sidvishnoi, @deniak, @marcoscaceres do you have any ideas why autopublishing to TR from the repo is failing — despite the fact the Status content that specberus is looking for is in fact in the spec?

The failures are at https://github.com/w3c/webappsec-secure-contexts/runs/3542602328#step:3:557

https://w3c.github.io/webappsec-secure-contexts/#status is the ED with the Status section included

**Comment by marcoscaceres** [comment]
Could it be that it's not wrapped in a `<section id="sotd">`?

**Comment by marcoscaceres** [comment]
Hmm... maybe not...

**Comment by deniak** [comment]
I think the issue comes from the fact that the whole TOC comes between the `<h2>Status of this Document</h2>` and all the paragraphs that section.

**Comment by deniak** [comment]
I'm not super familiar with bikeshed but the source document seems to be missing some closing `</p>`

File: index.bs [hunk 1]
```diff
@@ -67,6 +67,65 @@ urlPrefix: https://www.w3.org/2017/Process-20170301/; spec: W3C-PROCESS
...
+  is to have a minimum of two independent and interoperable user agents that
+  implementation all the features of this specification, which will be determined by
+  passing the user agent tests defined in the test suite developed by the Working
+  Group. The Working Group will prepare an implementation report to track progress.
+</p>
+
+<p>
+    This document was produced by a group operating under
+    the <a href="https://www.w3.org/Consortium/Patent-Policy-20200915/">W3C Patent Policy</a>.
+    W3C maintains a <a href="https://www.w3.org/groups/wg/das/ipr" rel=disclosure>public list of any patent disclosures</a>
```
Comment (xfq) [inline index.bs:117]: This should be https://www.w3.org/groups/wg/webappsec/ipr

**Comment by sidvishnoi** [comment]
To ease in debugging, I've added support for build artifacts in spec-prod (https://github.com/w3c/spec-prod/pull/102). Running the action now (in PR as well as merge) will create [an artifact with name `spec-prod-result`](https://user-images.githubusercontent.com/8426945/132528277-239a1f6c-ccbf-4eec-be76-b8e1dfcd01a5.png) containing output of the build which can be downloaded and inspected locally.

**Pull request merged**